upfadapter/pfcp/udp/transaction.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2024 Canonical Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// The content of this file was taken from https://github.com/omec-project/pfcp
```

Comment:
Do we need/want to keep this line?

Reply:
I added this here to make it clear that this code was not introduced by me but copied from the `pfcp` project.

Reply:
What about adding the copyright from the source file to this file? It could be a way to indicate that not all the code is original from Canonical. What do you think?

Reply:
Ok I've done that